windows-installer/docs/INSTALL.md

```diff
@@ -0,0 +1,85 @@
+# Certbot for Window (beta)
+
+The Certbot development team is proud to propose you the first beta release of Certbot for Windows.
+
+This document explains how to install Certbot and use it on Windows.
+
+## Supported features
+
+All usual operations to create and manage an account, manage existing certificates, or select the ACME server, are supported.
+
+In terms of authenticator plugins, `standalone`, `manual` and `webroot` are supported. DNS plugins will be available soon.
+
+For now, no installer plugins are available. `apache`, `nginx` will be available soon, and a new installer for IIS is under development.
+
+Certificate automated renewals are supported.
+
+All options available for a given supported command will work as described by the documentation in Certbot for Windows.
+
+## Installation
+
+The following operations needs to be executed using an account with **administrative privileges** on the machine you want to install Certbot.
+
+Certbot can be installed through an installer that is available:
+- on GitHub: go to the Certbot GitHub release (https://github.com/certbot/certbot/releases), select the last version of Certbot (v0.40.0 currently), and download the installer named `certbot-beta-installer-win32.exe in the assets.
+- on Certbot official website: _[specific URL and procedure to be defined]_
+
+The Certbot development team ensures the authenticity of the Windows installer only for these two sources.
```

Comment:
I would suggest "guarantees the authenticity" or "can guarantee the authenticity", and add

> Be cautious in downloading Windows installers from other sources, because they could be corrupt or include malicious functionality.